server/Changelog.md

```diff
@@ -9,6 +9,7 @@
 ### Enhancements
 
 - new cli command--`prefect-server dev build` will build prefect server images from source locally - [#2227](https://github.com/PrefectHQ/prefect/pull/2227)
+- Failing to set a state raises errors more aggressively - [#2708](https://github.com/PrefectHQ/prefect/pull/2708)
```

Comment:
The way we update the changelog has changed. Use the process outlined here https://github.com/PrefectHQ/prefect/tree/master/changes

Reply:
There's no new process for server?

Reply:
What we have been doing is moving the `server` section of the changelog up to the main changelog

Reply:
Ah sorry